src/System.Private.CoreLib/src/Internal/Reflection/Extensions/NonPortable/CustomAttributeSearcher.cs

```diff
@@ -41,7 +41,7 @@ public IEnumerable<CustomAttributeData> GetMatchingCustomAttributes(E element, T
             if (!skipTypeValidation)
             {
                 if (optionalAttributeTypeFilter == null)
-                    throw new ArgumentNullException("type");
+                    throw new ArgumentNullException(nameof(optionalAttributeTypeFilter));
```

Comment:
Helper that does validation for public APIs - the argument name in public APIs is not optionalAttributeTypeFilter. (Notice that the type is internal.)